client/templates/timeout.tmpl

```diff
@@ -3,6 +3,7 @@ import (
   "time"
 )
 
+{{$exclude := splitList "|" (index .Vars "exclude")}}
```

Comment:
I guess we're hitting the limitations of this templating

Reply:
Yes, but it's still useful to generate for most handlers.

Reply:
yeah, no concerns, just musing